Bottom: 32a4f1e57c822647aaea85798ab17269348c3ad7
Top:    c77e15879e929a7bcd9545d9e332138caa455fcb
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-14 20:17:12 +0000

Refresh of autoroute-c-remove-setting-the

---

diff --git a/src/autoroute.c b/src/autoroute.c
index 80aad25..ce75794 100644
--- a/src/autoroute.c
+++ b/src/autoroute.c
@@ -5183,7 +5183,6 @@ AutoRoute (bool selected)
     }
   if (PCB->Data->RatN == 0)
     return (false);
-  SaveFindFlag (DRCFLAG);
   rd = CreateRouteData ();
 
   if (1)
@@ -5357,7 +5356,6 @@ donerouting:
 
       Redraw ();
     }
-  RestoreFindFlag ();
 #if defined (ROUTE_DEBUG)
   aabort = 0;
 #endif
